### ISSUE TITLE
Wrong hint in the documentation for Docker variables

### ISSUE THREAD

**Issue by urbalazs** [WeblateOrg/weblate#10374]
### Describe the issue

In the documentation, there are wrong hints for the Docker variables:

- https://docs.weblate.org/en/latest/admin/config.html#gitea-credentials
- https://docs.weblate.org/en/latest/admin/config.html#gitlab-credentials

Both hints refers to `WEBLATE_GITHUB_*`.

### I already tried

- [X] I've read and searched [the documentation](https://docs.weblate.org/).
- [X] I've searched for similar issues in this repository.

### Steps to reproduce the behavior

1. Read the documentation.

### Expected behavior

The variables should be `WEBLATE_GITEA_*` and `WEBLATE_GITLAB_*`.

See:

- https://github.com/WeblateOrg/weblate/blob/main/docs/admin/install/docker.rst?plain=1#L790-L793
- https://github.com/WeblateOrg/weblate/blob/main/docs/admin/install/docker.rst?plain=1#L768-L771

### How do you run Weblate?

Docker container

**Comment by nijel** [comment]
Yes, the snippet is reused in all of them. Probably better to move it to Docker docs completely.
